eox_nelp/pearson_vue/api/v1/views.py

```diff
@@ -133,6 +136,30 @@ def create(self, request, *args, **kwargs):
 
         return response
 
+    def list(self, request, *args, **kwargs):
+        """
+        Retrieve a list of objects if the user has the necessary permissions.
+
+        This method overrides the default list method to add a permission check.
+        Only users who are superusers or staff members are allowed to retrieve the list of objects.
+        If the user does not have the necessary permissions, an HTTP 404 error is raised.
+
+        Args:
+            request (HttpRequest): The request object containing user information.
+            *args: Additional positional arguments.
+            **kwargs: Additional keyword arguments.
+
+        Returns:
+            Response: The response object containing the list of objects if the user has the necessary permissions.
+
+        Raises:
+            Http404: If the user does not have the necessary permissions.
+        """
+        if request.user.is_superuser or request.user.is_staff:
```

Comment:
So this force that the User used for the application to get the token, needs to be staff?
I think that only creating the application was enough to get a token and use the API.

Reply:
if you are not staff you shouldn't be able to get data from another users